wavefront_sdk/entities/histogram/histogram_impl.py

```diff
@@ -344,11 +329,10 @@ def get_max(self):
                 max_val = max(max(dist.percentile(100) for dist
                                   in minute_bin.per_thread_dist.values()),
                               max_val)
-        return max_val if max_val == max_val else None
+        return None if math.isnan(max_val) else max_val
```

Comment:
should use `float('-inf')`, `max()` cannot take `NaN`, it will remain `NaN`

Reply:
`float('NaN')` works fine with `max()`

```
>>> nan = float('nan')
>>> max(0.0, nan)
0.0
>>> max(9e99, nan)
9e+99
```

Reply:
```
>>> nan = float('nan')
>>> max(nan, 0.0)
nan
```
Guess the order matters, then let's just keep the original way